numbagg/moving.py

```diff
@@ -1,8 +1,50 @@
 import numpy as np
+from numba import guvectorize, float64
 
 from .decorators import ndmoving
 
 
+@guvectorize(
+    [(float64[:], float64, float64[:])],
+    '(n),()->(n)',
+    nopython=True,
+    target='parallel',
```

Comment:
My preference would be to setup a parallel import path for `target='parallel'` functions, e.g., `numbagg.parallel.ewm_nanmean()` or maybe `numbagg.ewm_nanmean.parallel()`. There are are uses for both types of gufuncs (e.g., depending on whether using dask or not).

But this is fine for now.

Reply:
That makes sense. 

Ideally we'd have one that worked well in each case (i.e. what is going to be  on `DataArray.ewm`?), but if there's a big difference in speeds then sobeit